### DISCUSSION TITLE
[Bug] do i need to update both server and client to 8.9.1 to avent log4j code injection?

### DISCUSSION THREAD

**Discussion by ligerman33** [apache/skywalking#8404] [Q&A]
### Search before asking

- [ ] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

now i am on 8.6.1 for java projects;
is it essential  update both server and client to 8.9.1 to avent log4j code injection?
do it means versions before 8.9.1 are under dangerous?

### What you expected to happen

now i am on 8.6.1 for java projects;
is it essential  update both server and client to 8.9.1 to avent log4j code injection?
do it means versions before 8.9.1 are under dangerous?

### How to reproduce

now i am on 8.6.1 for java projects;
is it essential  update both server and client to 8.9.1 to avent log4j code injection?
do it means versions before 8.9.1 are under dangerous?

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Why should this question be answered by a community?

**Reply by ligerman33** [reply]
i am just wonder whether shoud i update my skywalking to latest version

**Reply by wu-sheng** [reply]
That is totally your call. We are an open source community, not a service vendor. 
We are very clear about which versions are affected by log4j2 CVEs, and provided 8.9.1 as an emergency release patch. I think there is no more thing from our side.

And because you are asking about the client, I guess you mean agent. Java agent doesn't include log4j/log4j2 in any way. So it is always safe.

**Reply by ligerman33** [reply]
got it thank you!
